integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -38,6 +38,7 @@
  */
 public class Hooks {
 
+    private static final String WEBHOOKID = Utils.getSystemProperty("webhook.site.uuid");
```

Comment:
please use `WEBHOOK_ID` so the text is readable

Reply:
Ok, Done.